frontend/src/app/libs/common-components/models/tag.ts

```diff
@@ -0,0 +1,15 @@
+export class Tag {
```

Comment:
please avoid using classes, convert to interface since classes is hard objects for deepCopy which using during working with ngrx store.